<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskView</class>
 <widget class="QWidget" name="TaskView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="nameLayout">
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Название: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="resetNameButton">
       <property name="text">
        <string>Сбросить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="TextEdit" name="textEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>401</width>
        <height>174</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Стоимость:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="priceSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="minusButton">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="plusButton">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextEdit</class>
   <extends>QTextEdit</extends>
   <header>ModifiedClasses/TextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
